### ISSUE TITLE
deployment to firebase hosting 

### ISSUE THREAD

**Issue by PrestigeDevop** [blogifierdotnet/Blogifier#300]
Hi , I like this CMS from the demo , I've ran into some issue but in the end I  managed to get it to work on local server , 

however I tried to follow this article on how to deploy wasm into firebase hosting it doesn't work as expected :

https://swimburger.net/blog/dotnet/how-to-deploy-blazor-webassembly-to-firebase-hosting

**Comment by david-randoll** [comment]
Had any luck?

**Comment by PrestigeDevop** [comment]
> Had any luck?

No, it's probably possible but not feasible . There's new trend which's headless cms aka Jamstack. 

The article was demonstrate how to push wasm / automatic GitHub workflow trigger build. you probably could find a hacky way to do it though

**Comment by david-randoll** [comment]
Hey! I got Blogifier working on Aws with custom domains and an SSL certificate. It is using the AWS free tier so it's completely free. Check out the installation I write up using the Blogifier blog.

https://blog.davidrandoll.com/posts/host-blogifier-for-free-on-aws

**Comment by LuisThe0ne** [comment]
> > Had any luck?
> 
> No, it's probably possible but not feasible . There's new trend which's headless cms aka Jamstack. 
> 
> The article was demonstrate how to push wasm / automatic GitHub workflow trigger build. you probably could find a hacky way to do it though 

I think it is not possible, because firebase mainly supports static hosting and not all of Blogifier is written in blazor web assembly, I think only the admin panel. So it would be difficult or even impossible because you need to host it server side.
